bin/git-forgit

```diff
@@ -163,10 +164,7 @@ _forgit_add() {
     untracked=$(git config --get-color color.status.untracked red)
     # NOTE: paths listed by 'git status -su' mixed with quoted and unquoted style
     # remove indicators | remove original path for rename case | remove surrounding quotes
-    extract="
-        sed 's/^.*]  //' |
-        sed 's/.* -> //' |
-        sed -e 's/^\\\"//' -e 's/\\\"\$//'"
+    extract="sed 's/^.*]  //' | sed 's/.* -> //' | sed -e 's/^\\\"//' -e 's/\\\"\$//'"
```

Comment:
Just one note: this seems like an unrelated refactoring change. Should we remove this in order to keep the commit clean?

Reply:
Good catch! This change was made when I debug the keybind. Let's revert it.

Reply:
Done.